src/index.js

```diff
@@ -195,7 +195,10 @@ exports.init = (Implementation) => {
           'check on Forest that you copied it properly in the Forest ' +
           'initialization?');
       } else if (opts.envSecret) {
-        const collections = _.values(Schemas.schemas);
+        const forestadminSchemaFilename = `${path.resolve('.')}/forestadmin-schema.json`;
+        const collections = process.env.NOD_ENV === 'production'
+          ? JSON.parse(fs.readFileSync(forestadminSchemaFilename))
```

Comment:
We should print a clear message if:
- the file is invalid
- the file is empty
- the file is not present

Reply:
After this it's all good.